app/src/androidTest/java/ch/epfl/qedit/QuizActivityTest.java

```diff
@@ -5,14 +5,14 @@
 import static androidx.test.espresso.matcher.ViewMatchers.withId;
 import static androidx.test.espresso.matcher.ViewMatchers.withText;
 
-import androidx.test.ext.junit.runners.AndroidJUnit4;
+import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner;
```

Comment:
What is the advantage of using this instead of the old one?

Reply:
this is probably useful to Antoine's tests, i will ask him tomorrow

Reply:
Ok, I just couldn't find the answer right away by googling ...